core/src/main/java/org/apache/carbondata/core/index/IndexInputFormat.java

```diff
@@ -330,6 +339,11 @@ public void readFields(DataInput in) throws IOException {
         missingSISegments.add(in.readUTF());
       }
     }
+    this.isCDCJob = in.readBoolean();
```

Comment:
When `getSplits` will be called for index server, it will serialize `extendedBlockletWrapper`, so it will be written and deserialize wont be any problem.